src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +67,81 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={request}
+              onChange={(event) => {
+                setRequest(event.target.value);
+                setFailedRequest(false);
+              }}
```

Comment:
Create separate handler instead of anonymous function. Do it for all similar places